x/posts/internal/types/post_test.go

```diff
@@ -339,12 +349,47 @@ func TestPost_Validate(t *testing.T) {
 				},
 				date,
 				owner,
+				medias,
 			),
 			expError: "post optional data values cannot exceed 200 characters. key1 of post with id 1 is longer than this",
 		},
 		{
-			post:     types.NewPost(types.PostID(1), types.PostID(0), "Message", true, "Desmos", map[string]string{}, date, owner),
-			expError: "",
+			post: types.NewPost(
```

Comment:
I think this test and the following ones should be put inside another function as they verify the behavior of `PostMedias.Validate()` only

Reply:
similar tests are already inside post_media_test.go so if you want I can remove these two